addons/stock_account/models/account_move.py

```diff
@@ -121,8 +121,8 @@ def _stock_account_prepare_anglo_saxon_out_lines_vals(self):
                     .with_context(force_company=line.company_id.id)
                     .get_product_accounts(fiscal_pos=move.fiscal_position_id)
                 )
-                debit_interim_account = accounts['stock_output']
-                credit_expense_account = accounts['expense']
+                debit_interim_account = accounts['stock_output'] or self.journal_id.default_credit_account_id
+                credit_expense_account = accounts['expense'] or self.journal_id.default_debit_account_id
```

Comment:
These accounts are required on product category when dealing with automated stock valuation. Therefore, this diff isn't needed. Btw, this logic is not applied on `account.move.line` created by the `stock.move` and we expect both are creating entries using the same account. So, in my opinion, this is incorrect and not necessary.

Reply:
The expense account is not mandatory:
![Screenshot_2020-09-23 New - Odoo](https://user-images.githubusercontent.com/11041725/93989126-d4c13480-fd89-11ea-8bef-11bda3f3d955.png)

Reply:
Ok then. Also, you need to take `default_credit_account_id` in case of `out_refund`